### DISCUSSION TITLE
gsettings does not apply changes in dark theme

### DISCUSSION THREAD

**Discussion by libialany** [lennart-rth/Live-Earth-Wallpapers#16] [Ideas]
For ubuntu 22 Jammy gsettings does not apply changes in dark theme,I found a key that if it manages to apply :`picture-uri-dark`

**Comment by libialany** [comment]
can i contribute?

**Reply by lennart-rth** [reply]
I'm afraid this is already implemented but I did not test g settings as im using feh myself!
So feel free to make a Pull Request
